docs/assets/fhir-resource-ingestion-digestion-diagrams/bfd-pipeline-ccw-ingestion.plantuml

```diff
@@ -0,0 +1,32 @@
+@startuml
+title BFD-Pipeline-CCW Ingestion of Beneficiary/Claim Data Sequence
+participant ccw
+database s3
+participant "bfd-pipeline"
+participant splunk
+note over splunk: Specific Splunk Components: \n forwarders/indexers/search heads/ui/configuration
+participant slack
+database "bfd-db"
+ccw -> s3: Timestamped folder with: \n RIF files for a resource, manifest.xml \n placed in 'bfd-prod-etl' S3 bucket  
+s3 -> s3: Timestamped folder moved to \n 'Incoming' folder of bucket
+s3 -> splunk: New data discovered by splunk by quereying BFD-pipeline logs every 15 min
+splunk -> slack: Data discover status sent \n to #BFD-notices channel
+slack --> s3
```

Comment:
What's this step represent? In the diagram, it kinda looks like we're waiting for slack to respond before we continue, but the slack hook is basically a unidirectional, async notification that doesnt block our loading, right? Maybe its a limitation (or my misreading) of the flowchart, but its a tiny bit confusing.

Unless of course we actually do wait for some signal from the slack hook to start processing, in which case its fine!

Reply:
Yes its a unidirectional and async but there are multiple actors and so some steps have to be in slightly awkward chronological order. Like in the RDA version.